examples/examplesVersionDependency.ttl

```diff
@@ -66,6 +66,7 @@
 
 <https://w3id.org/semanticarts/ontology/gistCore>
 	a owl:Ontology ;
+	owl:versionIRI <https://w3id.org/semanticarts/ontology/gistCore12.2.2> ;
```

Comment:
Your other example on line 28 uses the unversioned import. Do you want to show that either are acceptable? Should we follow the best practice of not versioning the import IRI if there's a dependency assertion?

Reply:
I'm not 100% sure what we decided at today;s gist meeting.

Reply:
We decided that the best practice is to use a versioned IRI in the import and that it should be within the range specified in the dependencies.